src/services/users/index.js

```diff
@@ -152,11 +153,38 @@ const getNewTokens = async (email, currentRefreshToken) => {
 	return { accessToken, refreshToken, userId: user.id }
 }
 
+const getInvitedUserDetail = async (token) => {
+	try {
+		const decoded = jwt.verify(token, config('TOKEN_SECRET'))
+		const {email} = decoded.user
```

Comment:
@beblicarl, not sure this works because the when the data is being signed, it has no `user` object

https://github.com/tbf-challenge/teamwork-be/blob/ff64c32c1c59daa8f95bdf46430e78dc55fb9d49/src/services/users/index.js#L105

It has just email, so not sure you can get a `user` object from the decoded data.

Kindly double check and let me know if anything

Reply:
it actually has a user object with the email as being one of the properties